deps/build.jl

```diff
@@ -1,4 +1,5 @@
-import TimeZones: TZDATA_DIR, COMPILED_DIR
+using LightXML
```

Comment:
Include in any used packages in the REQUIRE file. In this case should be `@windows LightXML`